### ISSUE TITLE
Accessibility issue: The purpose of each link should be clear from the link text (5)

### ISSUE THREAD

**Issue by ameliejyc** [mineshmshah/mineshmshah.github.io#2]
See this link for more info:

https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_text_04

Applies to the following:

```
<a href=​"#t1">​
<li class=​"icon fa fa-smile-o" id=​"uno">​
::before​
</li>​
</a>​
<a href=​"#t2">​
<li class=​"icon fa fa-fast-forward" id=​"dos">​
::before​
</li>​
</a>​
<a href=​"#t3">​
<li class=​"icon fa fa-bolt" id=​"tres">​
::before​
</li>​
</a>​
<a href=​"#t4">​
<li class=​"icon fa fa-line-chart" id=​"cuatro">​
::before​
</li>​
</a>​
<a href=​"#t5">​
<li class=​"icon fa fa-thumbs-up" id=​"cinco">​
::before​
</li>​
</a>​
```
